AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -62,6 +62,7 @@ def camera(property: str = 'name') -> str:
         """
         properties = {
             'name': 'Camera',
+            'Field of view': 'Controller|Configuration|Field of view'
```

Comment:
here and for each new property: add a slight description to the docstring for each